plugins/labhub.py

```diff
@@ -90,12 +111,27 @@ def callback_message(self, msg):
         """Invite the user whose message includes the holy 'hello world'"""
         if not hasattr(self, 'TEAMS'):
             self.TEAMS = LabHub.get_teams(os.environ.get('GH_TOKEN'),
-                                          self.ORG_USERNAME)
+                                          self.GH_ORG_NAME)
 
         if re.search(r'hello\s*,?\s*world', msg.body, flags=re.IGNORECASE):
             user = msg.frm.nick
-            if not self.TEAMS[self.ORG_USERNAME + ' newcomers'].is_member(user):
+            if not self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user):
                 # send the invite
                 self.send(msg.frm,
                           self.INVITE_SUCCESS['newcomers'].format(user))
-                self.TEAMS[self.ORG_USERNAME + ' newcomers'].invite(user)
+                self.TEAMS[self.GH_ORG_NAME + ' newcomers'].invite(user)
+
+    @re_botcmd(pattern=r'(?:new|file) issue ([\w-]+?)(?: |\n)(.+?)(?:$|\n((?:.|\n)*))',  # Ignore LineLengthBear, PyCodeStyleBear
+               flags=re.IGNORECASE)
+    def create_issut_cmd(self, msg, match):
+        repo_name = match.group(1)
+        iss_title = match.group(2)
+        iss_description = match.group(3) if match.group(3) is not None else ''
```

Comment:
if the description is made in line after a space, then there is no way to differentiate if that is part of the title or part of the description